pages/about/privacy-policy.vue

```diff
@@ -1,9 +1,36 @@
 <template>
-    <p>PRIVACY POLICY</p>
+    <div class="px-64 py-24">
+        <h1 class="text-3xl my-8 text-center">2021 {{ $t('title') }}</h1>
+        <div v-for="content in $t('contents')" :key="`${content}`" class="my-4">
+            <h1 class="text-2xl my-8 text-center">
+                {{ content.title }}
+            </h1>
+            <p
+                v-for="item in content.description"
+                :key="`${item}`"
+                class="m-2"
+                v-html="item"
```

Comment:
As the warning suggested, please use [`i18n` tag](https://kazupon.github.io/vue-i18n/guide/interpolation.html#basic-usage) to avoid `v-html`.